app/javascript/components/Payment/Payment.js

```diff
@@ -443,14 +438,34 @@ export class Payment extends Component {
             />
           </Checkbox>
 
-          <DonateButton
-            currency={currency}
-            amount={donationAmount || 0}
-            submitting={this.state.submitting}
-            recurring={recurring}
-            disabled={this.disableSubmit()}
-            onClick={() => this.makePayment()}
-          />
+          {currentPaymentType === 'paypal' && (
```

Comment:
Can you explain this block of changes here? I don't see the reference to PayPal in the before diff - was the DonateButton segment all PayPal specific logic?